1-foundations/2-control-flows-and-functions/foundations-s2.ipynb

```diff
@@ -667,11 +1199,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 72,
+   "execution_count": 33,
    "id": "greek-gentleman",
    "metadata": {},
    "outputs": [],
    "source": [
+    "##  Creating a parametrized function (This is how your functions would look most of the times when implementing actual use cases)\n",
+    "\n",
     "def query_population(country = 'AFG', series = \"SP.POP.TOTL\", time = 2020):\n",
     "    \"\"\"\n",
     "    Function to retrieve indicator level data from WDI database. \n",
```

Comment:
Not sure what to request instead, but I wonder if this is too advanced. The participants learnt what a list and a float was the day before. While the API topic is very applicable for the audience and promotes WB resources, I wonder if this just risks throwing the participants off. 

Or maybe I there is something I am missing. Perhaps you did not mean for them to fully understand, just to give them an idea of what a real-life function might look like.

Reply:
As you keep using this function I think you should keep it in. But maybe make sure to make it clear that they are not expected to understand the details of an API. But the code should make some sense. What they really should be able to figure out is how to use it.

Reply:
I'll be sure to mention not to pay much attention with API but just string concatenation for now.